gutenberg.php

```diff
@@ -29,6 +29,13 @@ function the_gutenberg_project() {
 	global $post_type_object;
 	?>
 	<div class="block-editor gutenberg">
+		<noscript>
+			<div class="error" style='margin-top:32px'><p>
+			<?php
+			_e( 'The block editor requires Javascript to be enabled.', 'gutenberg' );
```

Comment:
JavaScript should be CamelCased!

```suggestion
			_e( 'The block editor requires JavaScript to be enabled.', 'gutenberg' );
```